### ISSUE TITLE
Default client password

### ISSUE THREAD

**Issue by jofmi** [directus-community/agency-os#36]
I had to manually set the password of `ashley@example.com` to `password` in order to be able to enter the client area.

**Comment by jbool24** [comment]
Are you hosting locally? I can't manage to get CORS to behave and let me into the portal. Wondering how you tested to get to login successfully (albeit having to change the hard-coded values in the LoginForm.vue component (that part is easy refactor for production use case) I just want to see the portal and haven't been able to get in.

**Comment by jofmi** [comment]
Hi @jbool24! Yes, I am hosting locally and experienced no problems with CORS. Often it appears as a CORS error if the backend is simply not reached. Have you checked that directus is running and available under the address that you put in your `.env` file?

**Comment by jbool24** [comment]
@jofmi yes confirming I am actually hitting the backend. All non-auth'd public pages are working. It's only on the `/portal` section I'm getting issues. In the browser console I can see the pre-flight requests and subsequent real POST request for the login failing with a CORS error. I used the default docker-compose inside '.directus' dir of this repo. I'm sure it's something easy I'm overlooking.

You don't have a proxy (on nuxt.config) setup for the front-end dev server, do you? You just hit 'http://localhost:{PORT}' on the front end app, right? Because I also tried setting up a local lookup with dnsmasq to make `example.test` resolve to localhost so that I could add 'http://example.test:3000' to the CORS_ORIGIN whitelist. That didn't make things work either.

**Comment by jofmi** [comment]
Very strange! Yes, I just hit localhost:3000.

**Comment by franforlizzi** [comment]
> @jofmi yes confirming I am actually hitting the backend. All non-auth'd public pages are working. It's only on the `/portal` section I'm getting issues. In the browser console I can see the pre-flight requests and subsequent real POST request for the login failing with a CORS error. I used the default docker-compose inside '.directus' dir of this repo. I'm sure it's something easy I'm overlooking.
> 
> You don't have a proxy (on nuxt.config) setup for the front-end dev server, do you? You just hit '[http://localhost:{PORT}](http://localhost:%7BPORT%7D)' on the front end app, right? Because I also tried setting up a local lookup with dnsmasq to make `example.test` resolve to localhost so that I could add 'http://example.test:3000' to the CORS_ORIGIN whitelist. That didn't make things work either.

same problem here!

**Comment by franforlizzi** [comment]
added
CORS_ENABLED: true
to docker-compose.yml and now it's working @jbool24

**Comment by jbool24** [comment]
@franforlizzi I had this ENV setting from the beginning. But it looks like it had more to do with docker volumes and caching I had in the DB. I did a fresh install of the whole stack and it worked correctly.

**Comment by jbool24** [comment]
So I've just added soft launched AgencyOS into my staging Directus instance and during testing most things seem to work ok. I've started customizing the "Blocks", pages, etc. But when I get back to the `/portal` related pages/functionality I'm getting the CORS issues again. The front-end is being worked on locally (obviously) and the backend is deployed in parity with what Production will look like (right now I'm using heroku).

The backend is located at a public domain such as: `https://cms.example.com`

I also have these ENV set on the Directus instance: in heroku:
```shell
CORS_ENABLED=true
CORS_ORIGIN=/^https:\/\/www.example.com$/,/^http:\/\/example.test:[0-9]{1,5}$/
```

Then on the nuxt app side, the production subdomain of the nuxt app will be @ `www` in PROD and then I also added a white-list entry in Directus for the testing local (as above) url which is resolved to `127.0.0.1` using dnsmasq locally on my machine.

When the portal page calls out to `/login` route directly for the login flow with Directus we get a CORS origin error.

@bryantgillespie a few questions:
1. Do you think this is just me and a user error; did I miss something?
2. Is that section of code trying to talk directly to Directus from the client?
3. If so, should the web client use its nuxt server routes for login flow, leaving all that logic in the nuxt side so that all that is same-origin?
